tests/integration_tests/test_upgrade.py

```diff
@@ -50,17 +64,16 @@ def test_upgrade(session_cloud: IntegrationCloud):
         return  # type checking doesn't understand that skip raises
 
     launch_kwargs = {
-        'name': 'integration-upgrade-test',
         'image_id': session_cloud._get_initial_image(),
-        'wait': True,
     }
 
     image = ImageSpecification.from_os_image()
 
     # Get the paths to write test logs
     output_dir = Path(session_cloud.settings.LOCAL_LOG_PATH)
     output_dir.mkdir(parents=True, exist_ok=True)
-    base_filename = 'test_upgrade_{os}_{{stage}}_{time}.log'.format(
```

Comment:
Is there an issue/etc for this change? Was the filename always wrong?

Reply:
It's the output log of the running test. It outputs some of the same commands before and after the upgrade and writes the output to before and after log files. When running tests in batch, getting a number of log files like:
```
test_upgrade_gce_focal_before_1.log
test_upgrade_ec2_focal_before_2.log
test_upgrade_lxd_focal_before_3.log
```
is much preferable to
```
test_upgrade_focal_before_1.log
test_upgrade_focal_before_2.log
test_upgrade_focal_before_3.log
```